src/reducers/alerts.js

```diff
@@ -19,6 +19,7 @@ const reducer = function (state, action) {
             const extension = extensionData.find(ext => ext.extensionId === extensionId);
             if (extension && extension.name) {
                 // TODO: is this the right place to assemble this message?
+                newAlert.extensionId = extensionId;
```

Comment:
Unrelated — typing out that previous comment made me wonder if `extensionId` can ever end up as `0`? Because then `if (extensionId)` above would be `false`...

Reply:
oh hm that's a good question, I am not sure and will look into it, thank you

Reply:
extensionId's are strings such as `ev3`, not numbers